### PR TITLE
Create two UnitTestBot plugin versions for IC and IU

### PR DESCRIPTION
**Pull request by EgorkaKulikov**
## Description

We decided two create two versions of our plugin:

1) For Intellij Ultimate: all supported features
2) For Intellij Community: Java + Spring + Python

## How to test

Automated tests: standard UTBot pipelines; note that yaml scripts were changed
Manual tests: some kind of regression for both plugin editions

## Self-check list

- [x] I've set the proper **labels** for my PR (at least, for category and component).
- [x] PR **title** and **description** are clear and intelligible.
- [x] I've added enough **comments** to my code, particularly in hard-to-understand areas.
- [x] The functionality I've repaired, changed or added is covered with **automated tests**.
- [ ] **Manual tests** have been provided optionally.
- [x] The **documentation** for the functionality I've been working on is up-to-date.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,15 +1,11 @@
 kotlin.code.style=official
 
 #Project Type
-# - PureJava:  no additional features
-# - ExperimentalLanguages: for Java + { Python, JavaScript, Go }
-# - Spring:   for Java + Spring
+# - Community:   for Java + Spring + Python (IC supported features)
 # - Ultimate: for all supported functionality
```
Comment (denis-fokin) [inline gradle.properties:5]: Let's mention all the functionality we support, like we did for the community.

**Comment by alisevych** [comment]
@denis-fokin @EgorkaKulikov @Markoutte 
Here are the two plugins generated from this branch:
https://github.com/UnitTestBot/UTBotJava/actions/runs/5819371184

**Review by Markoutte — APPROVED** [review]
LGTM. The only question I have is about how these both versions can be published, or the only one is going to be published? I think, this question should be discussed with @Vassiliy-Kudryashov

**Review by alisevych — COMMENTED** [review]
Please do NOT merge.
Testing in progress

**Comment by alisevych** [comment]
> @tyuldashev has found an issue with JS in IU plugin, investigating

The issue in JS test generation is old, not related to these changes:
- #2501 
PR Approved.

**Review by alisevych — APPROVED** [review]

**Pull request merged**